tests/plugins/colorbutton/colorbutton.js

```diff
@@ -190,7 +190,29 @@
 		'test text color items not draggable': testElementsNotDraggable( 'TextColor' ),
 
 		// (#2430)
-		'test background color items not draggable': testElementsNotDraggable( 'BGColor' )
+		'test background color items not draggable': testElementsNotDraggable( 'BGColor' ),
+
+		// (#4247)
+		'test panel should not have a scrollbar': function() {
```

Comment:
In fact you check here if the `overflow: hidden` is applied correctly, not the scrollbar itself, so maybe:

```suggestion
		'test panel should have styles that prevent scrollbar appearance': function() {
```